File: BlueSkyTaskThree - java version/src/main/java/com/bluesky/model/source/Branches.java
package com.bluesky.model.source;



import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Branches {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String name;
    private String description;
    private String city;
    private String state;
    private int postCode;
    private String country;
    private String contact;
    private String phone;


}

File: BlueSkyTaskThree - kotlin version/src/main/kotlin/com/bluesky/model/source/Branches.kt
package com.bluesky.model.source

import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
class Branches {


    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Int = 0
    var name: String = ""
    var description: String = ""
    var city: String = ""
    var state: String = ""
    var postCode: Int = 0
    var country: String = ""
    var contact: String = ""
    var phone: String = ""
}